src/scales/scale.category.js

```diff
@@ -49,22 +50,28 @@ module.exports = Scale.extend({
 	},
 
 	// Used to get data value locations.  Value can either be an index or a numerical value
-	getPixelForValue: function(value, index) {
+	getPixelForValue: function(value, index, datasetIndex) {
 		var me = this;
 		var offset = me.options.offset;
+
 		// 1 is added because we need the length but we have the indexes
 		var offsetAmt = Math.max((me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1)), 1);
 
+		var valueCategory, labels, idx;
+
+		if (index !== undefined && datasetIndex !== undefined) {
```

Comment:
Perhaps we should check using `helpers.isNullOrUndef` here instead of just checking for `undefined`. E.g. `getPixelForTick` was calling `return this.getPixelForValue(this.ticks[index], index + this.minIndex, null);` before this PR. If any plugins copied that code then they might break if we check only for `undefined`

Reply:
Good point. I have replaced them with `helpers.isNullOrUndef`.